vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_wallet.cc

```diff
@@ -132,10 +121,8 @@ void UpholdWallet::OnGenerate(
   }
 
   if (user.verified) {
-    uphold_->TransferAnonToExternalWallet(
-        std::move(wallet_ptr),
-        allow_zero_balance,
-        callback);
+    ledger_->TransferTokens(wallet_ptr->Clone(), [](const ledger::Result){});
```

Comment:
Is there a case where `TransferTokens` will get called twice in this function (at 108 and here)?

Reply:
removed